seedlet/functions.php

```diff
@@ -19,9 +19,8 @@
 /**
  * Determine whether the site is being requested from IE.
  */
-$is_ie = false;
-if ( preg_match( '~MSIE|Internet Explorer~i', $_SERVER['HTTP_USER_AGENT']) || (strpos($_SERVER['HTTP_USER_AGENT'], 'Trident/7.0; rv:11.0') !== false ) ) {
-	$is_ie = true;
+function seedlet_get_is_ie() {
```

Comment:
It turns out that WP core already declares `$is_IE` global that we should be able to reuse here.

https://github.com/WordPress/WordPress/blob/001ffe81fbec4438a9f594f330e18103d21fbcd7/wp-includes/vars.php#L93

The browser detection there seems to be more robust and a bit more performant since it uses `strpos` on `MSIE` instead of `preg_match` which is not really required (I mean, if we disregard the fact that it's already computed there 😄).

Reply:
That sounds very nice! Do we support win_ie or mac_ie?? :P

I'm in favor of that approach instead, but I'm not sure how to test it out in the theme

Reply:
TBH, looks like it just had a typo. AFAIK, PHP variables are case sensitive.
In this case rather than `$is_ie`, we should just change it to `$is_IE`, don't we?

Since these are being used in functions, you will need to add `global $is_IE;` as well.

I don't think we need to care about whether it's `win` or `mac`. It's used for IE specific stylesheet loading so it should apply to both win and mac. (that's what `$is_IE` doing)

Reply:
Good point, I will change this to point at `$is_IE`. I'm not sure that `$is_ie` was a typo since it is duplicating the same logic here, so it seems like it was just added as a way to check IE support without intending to rely on the logic in core